Pipfile

```diff
@@ -5,9 +5,13 @@ name = "pypi"
 
 [packages]
 django = "*"
+pytest = "*"
 
```

Comment:
We don't need it in packages
packages are for things we need only in a production environment